examples/with-cookie-auth/now.json

```diff
@@ -1,17 +1,5 @@
 {
   "version": 2,
   "name": "with-cookie-auth",
-  "build": {
-    "env": {
-      "API_URL": "http://localhost:3000"
-    }
-  },
-  "builds": [
-    { "src": "www/package.json", "use": "@now/next" },
-    { "src": "api/*.js", "use": "@now/node" }
-  ],
-  "routes": [
-    { "src": "/api/(.*)", "dest": "/api/$1" },
-    { "src": "/(.*)", "dest": "/www/$1" }
-  ]
+  "builds": [{ "src": "package.json", "use": "@now/next" }]
```

Comment:
We don't need to add the builder, only `name` and `version` are useful in this file, and also if you remove the entire file both `now` and `now dev` will work in the same way.

Reply:
Yeah, this whole file should just be deleted. 👍

Reply:
But if I use `now dev` without the `now.json` file locally, I will only see the folder structure in the browser.

![Next project folder structure](https://i.imgur.com/aViSTC5.png)

Reply:
I think it's not big deal, if it's a single app built with Next.js I should use `yarn dev`. I will delete it 👍

Reply:
@j0lv3r4 make sure you install the latest now-cli!